travertino/tests/colors/test_alpha_blending.py

```diff
@@ -1,7 +1,6 @@
 import re
 
 import pytest
-
```

Comment:
This implies travertino isn't being identified as first party...

Reply:
Ah - ok. I think the previous config make travertino first party *in travertino* but third party in Toga. If that's not possible to preserve, then we're probably better to make travertino first party across the board.

Reply:
To be clear, this triggers a significant number of changes. Is that your expectation?